bigtop-packages/src/charm/hadoop/layer-hadoop-resourcemanager/actions/smoke-test

```diff
@@ -43,6 +43,6 @@ smoke_env = {
 bigtop = Bigtop()
 result = bigtop.run_smoke_tests(smoke_components, smoke_env)
 if result == 'success':
-    hookenv.action_set({'outcome': result})
+    hookenv.action_set({'outcome': 'success'})
 else:
-    fail('{} smoke tests failed'.format(smoke_components), result)
+    fail('{} smoke tests failed with: {}'.format(smoke_components, result))
```

Comment:
Why combining the output and message here?  Seems like it would make the output harder to handle separately.  Is it just because the message is more important for CI?

I do like the consistent `outcome` field.